results/pkg/api/server/server.go

```diff
@@ -26,6 +28,15 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const (
+	listResultsDefaultPageSize int32 = 50
+	listResultsMaximumPageSize int32 = 10000
+)
+
+var uid = func() string {
+	return fmt.Sprintf("%v%s", time.Now().UnixNano(), uuid.New())
```

Comment:
uuid.New().String() is fine - UUIDs already include time information as a nonce.